docs/java-httpcomp/topics/client_auth.md

```diff
@@ -0,0 +1,27 @@
+To enable client authentication in the TLS handshake we only need to slightly modify the server authentication example. The instruction `socket.setNeedClientAuth(true)` will require the clients to present a valid certificate when connections are being accepted. The certificate validation will be based on the root CA certificates included in the `java.server.truststore.jks` file.
```

Comment:
I think each doc needs to be able to stand alone, so in this case, maybe briefly show how to create the trust store JKS "if you haven't already." You can see the kafka client auth doc for reference.

I think I also expect to see `java.server.truststore.jks` show up in the code sample below. Maybe this is the same thing.. it might be covered in the server auth doc, but we should reiterate here just in case they're only looking at client auth.